api/script/index.js

```diff
@@ -50,19 +51,27 @@ function conditionSend({
     splitAnalytic,
     ...params
 }) {
+    const conditionId = params.condition?.conditionId;
```

Comment:
Changes here are only for the new UI, which uses `rule.condition.*` methods, instead of `rule.rule.*`